src/api/Feed.js

```diff
@@ -705,13 +702,12 @@ class Feed extends Base {
         }
 
         task.task_assignment_collection.entries = assignments.map(taskAssignment => {
-            const { id, assigned_to, message, status } = taskAssignment;
+            const { id, assigned_to, status } = taskAssignment;
             return {
                 type: TASK_ASSIGNMENT,
                 id,
                 assigned_to,
-                message,
-                status: status.toLowerCase(),
+                status,
```

Comment:
The `toLowerCase` here might still be necessary, depending on how the API is updated.